llvm/test/MC/Disassembler/X86/sm4-evex-32.txt

```diff
@@ -0,0 +1,170 @@
+# RUN: llvm-mc --disassemble %s -triple=x86_64 | FileCheck %s --check-prefixes=ATT
```

Comment:
The file name is mistaken with sm4-evex-64.txt

Reply:
[23d4bfc](https://github.com/llvm/llvm-project/pull/113402/commits/23d4bfcabdca8968cba600e2fc0b440121990ef5)